source/2019-03-30-the-ember-times-issue-91.md

```diff
@@ -0,0 +1,138 @@
+---
+title: The Ember Times - Issue No. 91
+author: Anne-Greeth van Herwijnen, Chris Ng, Jessica Jordan, Bradley Leftley, Kenneth Larsen, the crowd
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
+alias : "blog/2019/03/30-the-ember-times-issue-91.html"
+responsive: true
+---
+
+හෙලෝ Emberistas! 🐹
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+<!--alex ignore hangman-hangwoman-->
+Read some RFCs around **Record Data Errors** 📇 and extending `EmberObject` 💡, learn how to build Hangman with **Ember Octane** 🎬 and about the **addon modules** RFC, check out the "e"mpressive EmberWeekend podcast with **@mansona**, ...
+
+
+...and read the exclusive interview with Ember contributor **@kategengler**.
+
+---
+
+## [Configuring Addon Modules in Module Unification Layout](https://github.com/ppcano/rfcs/blob/addon-module-config/text/0000-addon-module-config.md)
+
+A new RFC is out which proposes to provide an **API for addons** to configure their **module types** and collections for **Module Unification apps**. The reason for this is that at this moment Classic addons that require apps to locate classes into a non-default `app/` folder do not work "out of the box" in a Module Unification application.
+
+If you're looking to adopt Module Unification in the future, make sure to have a [look at the RFC](https://github.com/emberjs/rfcs/pull/462) and share your thoughts.
```

Comment:
And maybe also:

```suggestion
Since the feedback from early adopters of the still experimental Module Unification feature (MU) has been mixed so far, MU won't be in scope for the upcoming edition release of Ember Octane. This has also been outlined in the [recent, detailed update about the status of Module Unification](https://blog.emberjs.com/2019/03/11/update-on-module-unification-and-octane.html). Still, discussions about how a module unified future of Ember applications might look like are on-going and the RFC on the configuration of addon modules is another example of this conversation. If you want to learn more about how the design of addon integration in a future Module Unification world might look like, make sure to have a [look at the RFC](https://github.com/emberjs/rfcs/pull/462) and share your thoughts.
```